tests/model_registry/test_model_registry_creation.py

```diff
@@ -32,28 +31,51 @@ class TestModelRegistryCreation:
     # TODO: Switch to Python client
     @pytest.mark.smoke
     def test_registering_model(
-        self: Self,
-        model_registry_instance_rest_endpoint: str,
-        current_client_token: str,
-        updated_dsc_component_state,
+        self: Self, model_registry_instance_rest_endpoint: str, current_client_token: str, updated_dsc_component_state
```

Comment:
nitpick  `updated_dsc_component_state` is missing typing (i do not think that typing is needed in tests as they are not called directly / typing here can provide additional info)